cluster/vagrant/provision-master.sh

```diff
@@ -108,8 +104,8 @@ fi
 if ! which /usr/libexec/cockpit-ws &>/dev/null; then
 
   pushd /etc/yum.repos.d
-    wget https://copr.fedoraproject.org/coprs/sgallagh/cockpit-preview/repo/fedora-21/sgallagh-cockpit-preview-fedora-21.repo
-    yum install -y cockpit cockpit-kubernetes
+    wget https://copr.fedoraproject.org/coprs/sgallagh/cockpit-preview/repo/fedora-22/sgallagh-cockpit-preview-fedora-22.repo
```

Comment:
I had a problem with wget being missing on the fedora 23 libvirt image.  Do the opscode boxes include it by default?